mpl_config.py

```diff
@@ -48,6 +52,22 @@ def __init__(
         self.TEMP_W_IMG_DIR = self.ROOT_DIR + r"/data/water_mask/temp"
         self.OUTPUT_IMAGE_DIR = self.ROOT_DIR + r"/data/output_img"
         self.WORKER_ROOT = self.ROOT_DIR + r"/data"
+        self.MODEL_DIR = self.ROOT_DIR + r"/local_dir/datasets/logs"
```

Comment:
This looks like weird directory to have the model in...

Reply:
I agree, that's the location of it according to the maple code